src/components/generics/links/AddLink.vue

```diff
@@ -2,9 +2,13 @@
   <!-- We alawys display this link, even when user is not logged
     In this case, editions vues will redirect user to /auth -->
   <router-link :to="{name:documentType + '-add', params:{lang:$language.current}, query:query}">
-    <slot>
+    <template v-if="isShortLinkOnMobile">
```

Comment:
@brunobesson gosh Oo template inside template ?! what is this feature ?

Reply:
Well a template is only a way not to output a tag. Can't it be nested?

Reply:
I've learned something today, thank you very much. And actually, I remember that I did not succed to apply this logic, and defaulting on duplicated link with regrets. Good job :)